libfreefare/mifare_key_deriver.c

```diff
@@ -54,9 +54,10 @@ mifare_key_deriver_new_an10922(MifareDESFireKey master_key, MifareKeyType output
     }
 
     if ((deriver = malloc(sizeof(struct mifare_key_deriver)))) {
-	deriver->master_key = master_key;
-	deriver->output_key_type = output_key_type;
-	cmac_generate_subkeys(deriver->master_key);
+        deriver->master_key = master_key;
+        deriver->output_key_type = output_key_type;
+        cmac_generate_subkeys(deriver->master_key);
```

Comment:
The code use 4 spaces soft-tabs and 8 spaces tabs…  Quite unusual I admin… I guess `make style` will help with these :wink:

Reply:
Fixed.